css/style.css

```diff
@@ -194,10 +378,90 @@ p.content-description {
 .popular-content-item .item-img {
     align-items: flex-end;
 }
-.services-button-list li {    
+.services {
+    position: relative;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 205px;
+    max-width: 1440px;
+    height: auto;
+}
+.services-container {
     display: flex;
+    justify-content: flex-start;
     align-items: center;
+    width: 1160px;
+    margin-left: auto;
+    margin-right: auto;
+    height: auto;
+}
+.services::before {
```

Comment:
this can be just background with linear-gradient

Reply:
Yep. It can be. Done but now need to work with positioning